src/commcare_cloud/ansible/roles/formplayer/vars/main.yml

```diff
@@ -10,6 +10,7 @@ formplayer_build_url: "{{
 }}"
 formplayer_current_dir: "{{ formplayer_build_dir }}/current"
 formplayer_release_dir: '{{ formplayer_build_dir }}/releases/{{ new_release_name }}'
+formplayer_releases_dir: '{{ formplayer_build_dir }}/releases'
```

Comment:
I was tempted to rename `formplayer_release_dir` to `formplayer_current_release_dir`, but if there is a suggestion for a better name for `formplayer_releases_dir` open to hearing it to avoid potential confusion in the future.

Reply:
You could try `formplayer_releases_base_dir`, which emphasizes that it's the parent directory to each release dir